src/com/palmergames/bukkit/towny/db/TownyDatabaseHandler.java

```diff
@@ -139,10 +139,10 @@ public void finishTasks() {
 	public boolean backup() throws IOException {
 
 		if (!TownySettings.getSaveDatabase().equalsIgnoreCase("flatfile")) {
-			System.out.println("***** Warning *****");
-			System.out.println("***** Only Snapshots & Regen files in towny\\data\\ will be backed up!");
-			System.out.println("***** This does not include your residents/towns/nations.");
-			System.out.println("***** Make sure you have scheduled a backup in MySQL too!!!");
+			plugin.getLogger().warning("***** Warning *****");
+			plugin.getLogger().warning("***** Only Snapshots & Regen files in towny\\data\\ will be backed up!");
+			plugin.getLogger().warning("***** This does not include your residents/towns/nations.");
+			plugin.getLogger().warning("***** Make sure you have scheduled a backup in MySQL too!!!");
```

Comment:
Consider making this not actually warn, remain in white. We don't want to train people to ignore red/yellow during the towny startup.